src/schema/v2/notifications/index.ts

```diff
@@ -48,6 +49,24 @@ export const NotificationType = new GraphQLObjectType<any, ResolverContext>({
       resolve: ({ status }) => status === "unread",
     },
     createdAt: date(({ date }) => date),
+    formattedCreatedAt: {
```

Comment:
**question**: should we leave such a name or is it better `formattedPublishedAt`?

Reply:
Good question. I think we can use `publishedAt` instead. Maybe while you're in here, we can mark `createdAt` as deprecated and expose a new `publishedAt` field which matches the current implementation?

Reply:
Done

<img width="624" alt="image" src="https://user-images.githubusercontent.com/3513494/196173224-ec6a1b93-2ff9-4c29-811b-48c301c931e5.png">

Reply:
**question**: maybe it will be useful to provide an opportunity to query a formatted or raw date? for example, something like this

```graphql
{
  notificationsConnection(first: 10) {
    edges {
      node {
        publishedAt(formatted: true)
      }
    }
  }
}
```

Reply:
What do you both think of supporting both like:
```graphql
{
  notificationsConnection(first: 10) {
    edges {
      node {
        publishedAt // returns "2022-10-17T00:00:00+03:00"
        formattedPublishedAt // returns "Today", "Yesterday", "x days ago"
      }
    }
  }
}
```

Reply:
> I would prefer the first option (with an optional formatted: argument),

@gkartalis I'm also more prefer this

Reply:
feel free to go ahead with this, don't have any strong preference here

Reply:
Another option might be to extend the existing `date` field args which already support `format` and `timezone` args.

Maybe if `format === 'RELATIVE'`, we trigger the Today / Yesterday / N days ago format, otherwise we pass the format string (e.g. `YYYY-MM-DDTHH:mm:ss[Z]`) to moment as other fields do?

```graphql
{
  notificationsConnection(first: 10) {
    edges {
      node {
        publishedAt // "2022-10-17T00:00:00+03:00"
        publishedAt(format: "YYYY-MM-DD") // "2022-10-17"
        publishedAt(format: "RELATIVE") // "Yesterday"
      }
    }
  }
}
```

Reply:
@dblandin added changes

<img width="1145" alt="image" src="https://user-images.githubusercontent.com/3513494/196493972-2782909c-6b1e-48d2-8712-a3ee69a55877.png">